src/extension.ts

```diff
@@ -646,4 +351,47 @@ function createIntegrationsView(): void {
 			await camelKIntegrationsProvider.refresh().catch(err => console.log(err));
 		}
 	});
-}
\ No newline at end of file
+}
+
+export async function installDependencies() {
+
+	let gotKamel : boolean = false;
+	await checkKamelCLIVersion().then ( (kamelCliVersion) => {
+		if (kamelCliVersion) {
+			shareMessageInMainOutputChannel(`Found Apache Camel K CLI (kamel) version ${kamelCliVersion}...`);
+			gotKamel = true;
+		}
+	}).catch ( () => { 
+		// ignore 
+	});
+	const kubectlCliVersion : string = await checkKubectlCLIVersion();
+	if (kubectlCliVersion) {
+		shareMessageInMainOutputChannel(`Found Kubernetes CLI (kubectl) version ${kubectlCliVersion}...`);
+	}
+
+	const minikubeCliVersion : string = await checkMinikubeCLIVersion();
+	if (minikubeCliVersion) {
+		shareMessageInMainOutputChannel(`Found Minikube CLI (minikube) version ${minikubeCliVersion}...`);
+	}
+
+	if (!gotKamel) {
+		const installPromise = installDependency("kamel", gotKamel, installKamel);
```

Comment:
I think directly using await without intermediate variable is understandable and so woudl it make the code more concise.